intermediate-python-testing.md

```diff
@@ -361,3 +362,74 @@ Then, you need to patch the class you'd like to mock, and set its `return_value`
 mock_handle = mock.patch('path.to.CLASS_YOU_ARE_MOCKING')
 mock_handle.return_value = mocked_class
 ```
+
+## requests-mock
+
+Sometimes, your app sends requests to external URIs. And sometimes, these requests either have very little to do with the needs of your tests or fully interfere with them. [`requests-mock`](https://requests-mock.readthedocs.io/en/latest/overview.html) leaps into action!
```

Comment:
great points. i'd add that it violates test isolation to rely on requests to an external service!